### ISSUE TITLE
Conv2D channels_last in the Keras

### ISSUE THREAD

**Issue by cherishing78** [VinF/deer#57]
Hi Vince, I've noticed that the default mode of Conv2D function in the Keras is channels_last. In the 57th line and 73rd line of the NN_keras.py, the Reshape operation sets the channels as the first dimension. In my test, the function cannot work properly. The mountain_car_continuous works well because the dim == 1 and dim[0] == 1.

**Comment by VinF** [comment]
Hi, it should be fixed (PR #58).
